extras/kubernetes/README.md

```diff
@@ -33,7 +33,7 @@ Repeat as needed.
 First you will need to deploy the bootstrap Heketi container:
 
 ```
-$ kubectl create -f deploy-heketi-deployment.json
+$ kubectl create -f heketi-deployment.json
```

Comment:
It's correct that the deploy-heketi-deployment.json does not exist any more, but the whole text still talks about the bootstrap Heketi container. Somehow this text got broken in the last couple of updates...